chalice/cli/__init__.py

```diff
@@ -367,6 +372,31 @@ def generate_sdk(ctx, sdk_type, stage, outdir):
         raise click.Abort()
 
 
+@cli.command('generate-models')
+@click.option('--stage', default=DEFAULT_STAGE_NAME,
+              help="Chalice Stage for which to generate models.")
+@click.pass_context
+def generate_models(ctx, stage):
+    # type: (click.Context, str) -> None
+    factory = ctx.obj['factory']  # type: CLIFactory
+    config = factory.create_config_obj(stage)
+    graph_builder = ApplicationGraphBuilder()
+    graph = graph_builder.build(config, stage)
+    rest_apis = [m for m in list(graph.resources)
+                 if isinstance(m, models.RestAPI)]
+    if not rest_apis:
+        click.echo('No REST API found to generate model from.')
+        raise click.Abort()
+    swagger_generator = TemplatedSwaggerGenerator()
+    model = swagger_generator.generate_swagger(
+        config.chalice_app,
+        rest_apis[0]
```

Comment:
It looks like this rest api param is only needed for api gateway policies.  Can we simplify this and remove this param?